### DISCUSSION TITLE
Create a Dataset from Tensors

### DISCUSSION THREAD

**Discussion by vlomonaco** [ContinualAI/avalanche#549] [Feature Request] [answered]
At the moment we support the creation of **benchmarks** from tensors. However, it would be nice to support also the creation of a **dataset** from tensors. This is already done in a very shallow way by Pytorch with [TensorDataset](https://pytorch.org/docs/stable/_modules/torch/utils/data/dataset.html#TensorDataset). 

@lrzpellegrini this won't be supported by the current AvalancheDataset, am I right?

**Comment by lrzpellegrini** [comment, marked as answer]
We have AvalancheTensorDataset ;)

**Reply by vlomonaco** [reply]
ahah we definitely need to update the doc... these are useful features difficult to notice from the outside.
